cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -1404,71 +1404,60 @@ subroutine rectgrid
       if (my_task == master_task) then
          work_g1(:,:) = c0      ! initialize hm as land
 
-         if (trim(ew_boundary_type) == 'cyclic') then
+         if (trim(kmt_type) == 'boxislands') then
 
-            if (trim(kmt_type) == 'boxislands') then
+            call grid_boxislands_kmt(work_g1)
 
-               call grid_boxislands_kmt(work_g1)
+         elseif (trim(kmt_type) == 'channel') then
 
-            else ! default
-
-               do j = 3,ny_global-2      ! closed top and bottom
-               do i = 1,nx_global        ! open sides
-                  work_g1(i,j) = c1    ! NOTE nx_global > 5
-               enddo
-               enddo
-
-            endif ! kmt_type
-
-         elseif (trim(ew_boundary_type) == 'open') then
+            do j = 3,ny_global-2      ! closed top and bottom
+            do i = 1,nx_global        ! open sides
+               work_g1(i,j) = c1    ! NOTE nx_global > 5
+            enddo
+            enddo
 
-            if (trim(kmt_type) == 'boxislands') then
+         elseif (trim(kmt_type) == 'default') then
```

Comment:
I was under the impression that we were moving away from using `default` for the name of namelist options, maybe something more descriptive ? can't really think of a good name though...

Reply:
It's a balance of backwards compatibility and cleanup.  I lean towards backward compatibility as the baseline.  I always wonder if people update code and use "old" namelists.  Maybe they don't, but if they do, it'll cause problems.  Maybe we can keep default as an undocumented option but change the documented setting to something else and then support both options as the same.

Reply:
Yes, this has bitten me in the past. I agree then that keeping backward compatibility is better. Thanks!